core/benchmarks/bm_Buffer.cpp

```diff
@@ -103,7 +103,7 @@ testNewAPI(Buffer auto &buffer, const std::size_t vector_length, const std::size
                     const auto &input = reader.get(vector_length);
```

Comment:
maybe rename `input`  to `consumableInputSpan` or name the type `ConsumableSpan auto`....